scripts/pnodespy.py

```diff
@@ -30,18 +30,27 @@ class {0}(IntEnum):
         print("    {0} = {1}".format(k, n))
 
 
-def print_file_header():
+def print_file_header(includeIntEnumunique=True,includeBindToLibGHDL=True):
     print(dedent("""\
             # Auto generated Python source file from Ada sources
             # Call 'make' in 'src/vhdl' to regenerate:
             #
-            from enum import IntEnum, unique
-            from pydecor import export
-
-            from pyGHDL.libghdl._decorator import BindToLibGHDL
         """), end=''
     )
 
+    if includeIntEnumunique:
```

Comment:
I think it's better to use this form:

```py
print(dedent("""\
    # Auto generated Python source file from Ada sources
    # Call 'make' in 'src/vhdl' to regenerate:
    #
    {sysImports}
    
    from pydecor import export
    
    {moduleImports}
    """.format(
        sysImports = "from enum import IntEnum, unique" if includeIntEnumunique else "",
        moduleImports  "from pyGHDL.libghdl._decorator import BindToLibGHDL" if includeBindToLibGHDL else "",
    )
))
```

This doesn't break the reading flow so much.

Reply:
Done.